test/manual/tls_support.test.ts

```diff
@@ -106,6 +108,42 @@ describe('TLS Support', function () {
       });
     });
   });
+
+  context('when tlsCertificateKeyFile is provided, but tlsCAFile is missing', () => {
+    let client: MongoClient;
+    beforeEach(() => {
+      client = new MongoClient(CONNECTION_STRING, {
+        tls: true,
+        tlsCertificateKeyFile: TLS_CERT_KEY_FILE
+      });
+    });
+    afterEach(async () => {
+      if (client) await client.close();
+    });
+
+    it('throws an error', async () => {
+      const err = await client.connect().catch(e => e);
+      expect(err).to.be.instanceOf(Error);
+    });
+  });
+
+  context('when tlsCAFile is provided, but tlsCertificateKeyFile is missing', () => {
```

Comment:
I actually think it should work when passing the CA but not the key file. I'm looking at a Python test that is expected to succeed in this case. https://github.com/mongodb/mongo-python-driver/blob/5bd444a604f85de082646322263031c98af8c213/test/test_ssl.py#L380

Reply:
Do we know why that's the case? I was trying to find information on that yesterday, but didn't have any luck.

Reply:
You should always be able to specify a CA file when using TLS, yes. The CA file would be used to validate the server certificate; `tlsCertificateKeyFile` is something that is specific to X.509 authentication, but `tlsCAFile` isn’t.